finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ssl/client/Netty4ClientSslConfigurations.scala

```diff
@@ -59,6 +59,8 @@ private[finagle] object Netty4ClientSslConfigurations {
           cert <- new X509CertificateFile(certFile).readX509Certificate()
           chain <- new X509CertificateFile(chainFile).readX509Certificates()
         } yield builder.keyManager(key, cert +: chain: _*)
+      case KeyCredentials.KeyManagerFactory(keyManagerFactory)          =>
```

Comment:
Can you add a note to the comments for this method that indicates that unlike Cert/Key Cert/Key/Chain, that using KeyManagerFactory does not ensure that the certificates contained within the KeyManagerFactory are not expired.